src/components/ColorCombos/index.js

```diff
@@ -22,6 +22,8 @@ export default class ColorCombos extends Component {
                 className="row combo-row"
                 key={`${fg.forced || fg.originalIndex}-${bg.forced ||
                   bg.originalIndex}`}
+                tabIndex={-1}
+                ref={el => (this.listItem = el)}
```

Comment:
Should use an instance method to avoid creating this closure over and over agian.